src/server/lib/templateRules.json

```diff
@@ -3800,7 +3878,7 @@
       "index": 16,
       "required": true,
       "dataType": "Numeric",
-      "maxLength": 6,
+      "maxLength": 10,
```

Comment:
you used to only be able to rehire 999,999 FTEs, but now I guess you can go up to 9,999,999,999 . that's larger than the population of the world, so we shouldn't need to increase this maxlength again.

unless, of course, they expect the commas. i'm not really sure what `Numeric` means, but i've been burned ever since the subrecipient `uei` was numeric -- length 10, and that meant a format like `\d{2}-\d{7}`

Reply:
Treasury does NOT want commas. There was actually no max length listed, so I just made it consistent with other numeric fields. Call it future proofing.